wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -162,22 +164,22 @@ <h2 class="usa-accordion__heading">
                         <div id="ts-a1" class="usa-accordion__content">
                             <table class="usa-table" id="site-data-summary">
                                 <thead>
-                                    <tr>
-                                        <th scope="col">USGS Parameter Group</th>
-                                        <th scope="col">Data Types</th>
-                                        <th scope="col">Start Date</th>
-                                        <th scope="col">End Date</th>
-                                    </tr>
+                                <tr>
```

Comment:
Nit: This should be indented.

Reply:
Oh, sorry, I cut an replaced the page and the the IDE decided that this was the way to go. lol. I'll change it.